newrelic/newrelic-browser-agent
# NR-73497: Removing phantomjs

### Overview

Removing all phantomjs and all references to it.

### Related Github and/or JIRA Issue(s)

https://issues.newrelic.com/browse/NR-73497

### Testing

1. Pull the branch
2. Rlean all the `node_module` directories
3. Run `npm i`
4. Run `npm ls phantom`
5. Verify the dependency is not found in the project

All the tests should also pass with the `safe to test` label.
